QUDAConfig.cmake.in

```diff
@@ -7,6 +7,10 @@ set(QUDA_MPI @QUDA_MPI@)
 set(QUDA_QIO @QUDA_QIO@)
 set(QUDA_OPENMP @QUDA_OPENMP@)
 set(QUDA_QDPJIT @QUDA_QDPJIT@)
+set(QUDA_VERSION @CMAKE_PROJECT_VERSION@)
+set(GITVERSION @GITVERSION@)
```

Comment:
I guess we probably should rename this to be prefixed with `QUDA_` if we expose this.

Reply:
Sure! FYI I need it for checking if a tunecache became obsolete